### ISSUE TITLE
Adding layers

### ISSUE THREAD

**Issue by amobiny** [naturomics/CapsNet-Tensorflow#54]
Thanks for the nice code. I had a question regarding the capsnet. Would it be possible to add layers (like a conv-caps layer after the first primary layer, or a fully connected caps layer with 20 capsules befor the digit caps layer?)
I tried it myself and am getting terrible results! I can't understand why is it happening. Do you have any idea?

**Comment by naturomics** [comment]
Yes, I created a new project to meet your situation. Please see my [CapsLayer](https://github.com/naturomics/CapsLayer) repo. You can `import capslayer` and then call it in your code: `capslayer.layers.conv2d(...)` or `capslayer.layers.fully_connected(...)`
